firebase-appdistribution/test-app/src/main/java/com/googletest/firebase/appdistribution/testapp/NotificationFeedbackTrigger.kt

```diff
@@ -35,7 +35,9 @@ object NotificationFeedbackTrigger : Application.ActivityLifecycleCallbacks {
   private var isEnabled = false
   private var hasRequestedPermission = false
   private var currentActivity: Activity? = null
-  private var requestPermissionLauncher: ActivityResultLauncher<String>? = null
+  // TODO(lkellogg): this is getting too complex - simplify it by, for example, only enabling this
+  //   on a per-activity basis instead of app-wide
+  private var requestPermissionLaunchers: MutableMap<Activity, ActivityResultLauncher<String>?> = mutableMapOf()
```

Comment:
should this be a `WeakHashMap`? I don't think we want to prevent garbage collection of `Activities`.